xtable-core/src/main/java/org/apache/xtable/delta/DeltaConversionTarget.java

```diff
@@ -264,7 +330,8 @@ private void commitTransaction() {
       transaction.updateMetadata(metadata, false);
       transaction.commit(
           actions,
-          new DeltaOperations.Update(Option.apply(Literal.fromObject("xtable-delta-sync"))));
+          new DeltaOperations.Update(Option.apply(Literal.fromObject("xtable-delta-sync"))),
+          ScalaUtils.convertJavaMapToScala(getCommitTags()));
```

Comment:
What is the difference between putting this info in tags compared to the metadata we are currently using? Should we consolidate?

Reply:
Here `xtable-delta-sync` is the operation, and the tags is more recommended to store the metadata as k-v pairs